arch/arm/soc/atmel_sam/sam4s/Kconfig.soc

```diff
@@ -112,7 +105,15 @@ config SOC_ATMEL_SAM4S_PLLB_DIVB
 	Board config file can override this settings
 	for a particular board.
 
-	With default of MULB == 6, and DIVB == 1,
-	PLL is running at 7 times of main clock.
+	With default of MULB == 3, and DIVB == 1,
+	PLL is running at 4 times of main clock.
+
+config SOC_ATMEL_SAM4S_EFC0_FMR_FWS
```

Comment:
If we decide that we need to ask user to provide VDDCORE and VDDIO to support run time frequency modifications then this option becomes redundant. If we decide that the only way to support run time frequency modifications is to have user explicitly provide FWS value then this option would be fine. I would rather leave this discussion for some later time.